package.json

```diff
@@ -31,12 +31,15 @@
     "typescript": "^4.2.4"
   },
   "dependencies": {
+    "@cosmjs/amino": "^0.29.4",
+    "@cosmjs/crypto": "0.28.13",
     "@cosmjs/encoding": "^0.28.13",
     "@cosmjs/proto-signing": "^0.28.13",
     "@cosmjs/stargate": "^0.28.13",
     "@cosmjs/tendermint-rpc": "^0.28.13",
+    "@osmonauts/lcd": "^0.8.0",
     "cosmjs-types": "^0.5.1",
     "google-protobuf": "^3.17.0",
-    "long": "^4.0.0"
```

Comment:
we are importing long in `src/proto/index.tx` and should keep explicit import. version 5.2.1 is current if we want to update